views/documentation_word.rhtml

```diff
@@ -0,0 +1,440 @@
+<!DOCTYPE html>
```

Comment:
What do you think about next:
I propose to split this file by the sections and create the main file as an entry point - where all our partials will be rendered